src/Resources/config/services.yml

```diff
@@ -128,10 +128,10 @@ services:
     cmf.activity_store:
         alias: CustomerManagementFrameworkBundle\ActivityStore\ActivityStoreInterface
 
-    CustomerManagementFrameworkBundle\ActivityStore\ActivityManagerInterface:
-        class: CustomerManagementFrameworkBundle\ActivityManager\DefaultActivityManager
-    cmf.activity_manager:
-        alias: CustomerManagementFrameworkBundle\ActivityStore\ActivityManagerInterface
+    cmf.activity_manager: '@CustomerManagementFrameworkBundle\ActivityManager\ActivityManagerInterface'
+    CustomerManagementFrameworkBundle\ActivityStore\ActivityManagerInterface: '@CustomerManagementFrameworkBundle\ActivityManager\ActivityManagerInterface'
+    CustomerManagementFrameworkBundle\ActivityManager\ActivityManagerInterface: '@CustomerManagementFrameworkBundle\ActivityManager\DefaultActivityManager'
+    CustomerManagementFrameworkBundle\ActivityManager\DefaultActivityManager: ~
```

Comment:
Not really sure about this one - @markus-moser could you have a look? Currently there are definitions for `CustomerManagementFrameworkBundle\ActivityStore\ActivityManagerInterface` and `CustomerManagementFrameworkBundle\ActivityManager\ActivityManagerInterface`, but the former class does not exist.

Reply:
any updates on that one?

Reply:
`CustomerManagementFrameworkBundle\ActivityStore\ActivityManagerInterface` is a completly useless service - must be a copy/paste error. I removed it in this pull request.
Are there any other questions?

Reply:
Nope, let's merge :)